### DISCUSSION TITLE
how to login getoutline cloud service?

### DISCUSSION THREAD

**Discussion by weironz** [outline/outline#8097] [General] [answered]
![image](https://github.com/user-attachments/assets/b531def2-b366-4cad-b2b6-d529980806dd)

login
![image](https://github.com/user-attachments/assets/aa2fcad5-9825-444f-ad34-e7f57bc11ba0)

login

![image](https://github.com/user-attachments/assets/58f8ecbb-2aa9-48f0-9230-b91163fcd312)

login
![image](https://github.com/user-attachments/assets/ba013528-97c6-4625-b67c-243f4587b1ca)

faild
![image](https://github.com/user-attachments/assets/22f82ff7-955b-4048-8436-6ab60a2bcf1a)

It always say:
"Failed to log in. Please navigate to the custom URL of your workspace and try to log in again.

If you have been invited to join a workspace, you will find a link to that workspace in the invitation email."

**Comment by tommoor** [comment, marked as answer]
Please email `hello@getoutline.com` for questions regarding the hosted service, all I can reveal in public is that the login domain is incorrect
